src/icesat2_tracks/local_modules/m_general_ph3.py

```diff
@@ -474,13 +469,13 @@ def power_imshow(
                 dd2 = ddn
                 fn = self.fs[fsn_p]
 
-                if nopower is True:
+                if nopower:
                     tt = tt
                 else:
                     tt = tt[:-1]
```

Comment:
I just noticed this can be simplified to:
```suggestion
                if not nopower:
                    tt = tt[:-1]
```